2_Operations/work-process.md

```diff
@@ -16,11 +16,10 @@ Sprint meetings replace the daily standup on these days:
    * 10:00 CE(S)T for the whole company in the Niteo Zoom room
    * followed immediately by separate plannings for Easy Blog Networks and WooCart, in their own Zoom rooms
    * followed by [Developers Session](#developers-session)
- * The **Tuesday**, two weeks later, marks the end of the sprint with a sprint retrospective and company alignment meeting.
-   * 10:00 CE(S)T for Niteo
-   * followed immediately by retrospective for Easy Blog Networks 
-   * 11:00 CE(S)T for WooCart
-   * 12:00 CE(S)T for Niteo Alignment Meeting
+ * The **Tuesday**, two weeks later, marks the end of the sprint with a sprint retrospective.
+   * 10:00 CE(S)T for EBN
+   * followed immediately by retrospective for WooCart
+   * 11:00 CE(S)T for Niteo
```

Comment:
We usually need more than one hour for Retros, especially for two Retros. Especially now that alignment is done in Retro too.